randovania/games/dread/json_data/Dairon.txt

```diff
@@ -288,7 +311,9 @@ Extra - asset_id: collision_camera_004
   > Pickup (Missile Tank)
       All of the following:
           Morph Ball
-          Grapple Beam or Space Jump or Speed Booster or Simple IBJ
+          Any of the following:
+              Grapple Beam or Space Jump or Speed Booster or Simple IBJ
+              Flash Shift and Use Spin Boost
```

Comment:
I feel like you could also add a cross bomb alternative:

Flash Shift AND (Use Spin Boost OR Lay Cross Comb)

Reply:
i would need to test to see if that is doable. you wouldnt get any horizontal movement from the cross bomb